src/keysBuilder/buildTranslationFile.ts

```diff
@@ -10,8 +11,13 @@ export type FileAction = {
   type: 'new' | 'modified';
 };
 
-export function buildTranslationFile(path: string, translation = {}, replace = false): FileAction {
-  const currentTranslation = fsExtra.readJsonSync(path, { throws: false }) || {};
+export function buildTranslationFile(
+  path: string,
+  translation = {},
+  replace = false,
+  deleteMissing = false
```

Comment:
If it's in the default config, do we really need a default value?
Also, I think it would e better to pass an options object to `buildTranslationFile` instead of all these params.

Reply:
I think it was added here because there is a call to this method from `webpack-plugin.ts` without the `replace` or `deleteMissing` arguments.